.changelogs/llms-sendwp-do-remote-install.yml

```
@@ -0,0 +1,3 @@
+significance: major
+type: changed
+entry: Changed {@see LLMS_SendWP::do_remote_install()} access from public to protected.
```

```suggestion
entry: Changed `LLMS_SendWP::do_remote_install()`.
```

I don't want docblock markup that in user-facing changelogs. Mostly no one reading this will do so within an IDE. Markdown formatting is used because (most) places where the changelog is displayed, the markdown formatting is parsed.